src/ecs/systems/rendering.rs

```diff
@@ -54,19 +61,47 @@ impl<'a> System<'a> for RenderSystem {
     }
 }
 
-impl RenderSystem {
+impl<'a, 'b> SystemExt<'a, (&'b EventsLoop, PipelineBuilder)> for RenderSystem {
```

Comment:
I don't see how the `SystemExt` trait is useful; one can just make an associated method which has the advantage that we don't need to include a trait, don't have to use a tuple for the real arguments and don't have any restrictions wrt return type, number of flavors,...